Source/Fuse.Controls.CameraView/Example/CameraViewExample/.unoconfig_example

```diff
@@ -0,0 +1 @@
+Packages.SourcePaths += /www/github/fuselibs/Source
```

Comment:
This file looks like a mistake. I doubt that path is useful on other people's machines ;)

Reply:
that's why its an example :D

Reply:
...plus its great for testing the example with latest source

Reply:
I don't understand. This path is only useful on your computer. Put it somewhere else, and you should be fine without adding noise like this...

Reply:
Its useful for the testing of new source workflow, which you guys haven't mentioned; after pulling a change to the libs or making a change, the project doesn't include the updated source files without using the .unoconfig file and "uno doctor" command on a project. The fuse libs work the first time when following the instructions but not again thereafter. This is for mac: https://forums.fusetools.com/t/run-fuselibs-locally/7250